std/uuid.d

```diff
@@ -1203,10 +1207,32 @@ public struct UUID
  *
  * This function is not supported at compile time.
  *
- * Params:
- *      randomGen = uniform RNG
+ * Warning:
+ * This function should not be used when UUIDs need to be cryptographically secure,
+ * as the output of this function can be predicted.
+ *
+ * Deprecated:
+ * The `randomUUID()` overload not taking any arguments has been deprecated.
+ * It is dangerous, as it causes predictable, non-secure UUIDs and has not been
+ * documented as doing that previously. Silently changing this function to use a secure
+ * system RNG is not possible, as these RNGs may block at early boot when they are
+ * not yet seeded, causing `randomUUID` to block as well.
+ * See $(LINK2 https://forum.dlang.org/thread/zjmyxudvscfwrkrdabmt@forum.dlang.org, this forum thread)
+ * for a detailed discussion of the problem.
+ *
+ * Depending on whether the desired output needs to be cryptographically secure and
+ * considering the potential blocking problem, users should use `rndGen` (insecure)
+ * from std.random or a secure alternative like this:
+ * ----------------------------
+ * auto insecureUUID = rndGen.randomUUID();
+ * ----------------------------
+ *
  * See_Also: $(REF isUniformRNG, std,random)
  */
+deprecated("randomUUID() generated UUIDs are not suited for cryptographic use. " ~
+    "As the default cannot be changed without silent breakage, it must now be explicitly specified " ~
+    "whether cryptographically secure output is required. See randomUUID() documentation " ~
+    "for more details.")
```

Comment:
I think it would be better if this wasn't such a long text + if it would tell me easier how to update my code.

I think something like `` `randomUUID()` is cryptographically insecure, use `rndGen.randomUUID()` if this was intended`` brings across the message a lot more clearly and is also easier to read. (especially for non-native speakers)

I think people who do want something cryptographically secure would then naturally consult the documentation.